src/main/java/org/fundacion/jala/converter/models/parameter/JavaParameter.java

```diff
@@ -5,25 +5,28 @@
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
- */
-/**
- * @author Paola Aguilar
+ *
+ * @author Paola Ximena Aguilar Quiñones
  */
 package org.fundacion.jala.converter.models.parameter;
 
 import org.fundacion.jala.converter.service.javacompiler.JavaVersion;
 
+/**
+ * This class is the options of the java parameters.
```

Comment:
A better description for this class, could be, This class has/show/manage the options for the Java parameters.

Reply:
It was solved